crates/biome_css_analyze/tests/specs/nursery/noDuplicateProperties/invalid.css.snap

```diff
@@ -49,337 +49,408 @@ a { color: red !important; color: blue; }
-  
+
 
 ```
 
 ```
 invalid.css:42:28 lint/nursery/noDuplicateProperties ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
   ! Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.
-  
+
     40 │ a { -webkit-border-radius: 12px; -webkit-border-radius: 10px; }
-    41 │ 
+    41 │
   > 42 │ a { color: red !important; color: blue; }
        │                            ^^^^^
-    43 │ 
+    43 │
     44 │ a { color: red !important; color: blue !important; }
-  
+
   i color is already defined here.
-  
+
     40 │ a { -webkit-border-radius: 12px; -webkit-border-radius: 10px; }
-    41 │ 
+    41 │
   > 42 │ a { color: red !important; color: blue; }
        │     ^^^^^
-    43 │ 
+    43 │
     44 │ a { color: red !important; color: blue !important; }
-  
+
   i Remove or rename the duplicate property to ensure consistent styling.
-  
+
 
 ```
 
 ```
 invalid.css:44:28 lint/nursery/noDuplicateProperties ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
   ! Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.
-  
+
     42 │ a { color: red !important; color: blue; }
-    43 │ 
+    43 │
   > 44 │ a { color: red !important; color: blue !important; }
        │                            ^^^^^
-    45 │ 
-  
+    45 │
+    46 │ a {
+
   i color is already defined here.
-  
+
     42 │ a { color: red !important; color: blue; }
-    43 │ 
+    43 │
   > 44 │ a { color: red !important; color: blue !important; }
        │     ^^^^^
-    45 │ 
-  
+    45 │
+    46 │ a {
+
+  i Remove or rename the duplicate property to ensure consistent styling.
+
+
+```
+
+```
+invalid.css:49:9 lint/nursery/noDuplicateProperties ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.
+
+    47 │     @supports (color: pink) {
+    48 │         color: pink;
+  > 49 │         color: orange;
+       │         ^^^^^
+    50 │     }
+    51 │ }
+
+  i color is already defined here.
+
+    46 │ a {
+    47 │     @supports (color: pink) {
+  > 48 │         color: pink;
+       │         ^^^^^
+    49 │         color: orange;
+    50 │     }
+
+  i Remove or rename the duplicate property to ensure consistent styling.
+
+
+```
+
+```
+invalid.css:58:13 lint/nursery/noDuplicateProperties ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.
+
+    56 │         &:hover {
+    57 │             color: orange;
+  > 58 │             color: black;
+       │             ^^^^^
+    59 │         }
+    60 │     }
+
+  i color is already defined here.
+
+    55 │         color: pink;
+    56 │         &:hover {
+  > 57 │             color: orange;
+       │             ^^^^^
+    58 │             color: black;
+    59 │         }
```

Comment:
it seems that the lint rule properly throws errors for duplicated CSS properties